.github/workflows/publish-release.yml

```diff
@@ -43,9 +43,16 @@ jobs:
           if [[ -z "${RELEASE_BODY}" && -f "CHANGELOG.md" ]]; then
             RELEASE_BODY="See [CHANGELOG.md](CHANGELOG.md) for changes details."
           fi
+          # Determine if the release is a prerelease
```

Comment:
Not sure if the comment will be present in the script or considered as a yaml comment

```suggestion
```